pillar/fluentd/mitx.sls

```diff
@@ -7,9 +7,8 @@ fluentd:
       - ruby2.3
       - ruby2.3-dev
       - build-essential
-  plugins:
-    - fluent-plugin-secure-forward
   configs:
+    - '@include': 'fluentd_log'
```

Comment:
This won't work at the top level of the `configs` object. If we want it included for all fluentd configs we would add it to the `init.sls` file, otherwise we would need to move it down into the `settings:` object under the edx config.